share/models/validators.py

```diff
@@ -3,29 +3,15 @@
 import ujson
 from collections import OrderedDict
 
-import regex
-
 from jsonschema import exceptions
-from jsonschema import Draft4Validator
+from jsonschema import Draft4Validator, draft4_format_checker
 
 from django.db import connection
 from django.apps import apps
 from django.core.exceptions import ValidationError
 from django.utils.deconstruct import deconstructible
 
-
-def is_valid_uri(value):
-    # uri = rfc3987.get_compiled_pattern('^%(URI)s$')
-    try:
-        assert regex.match(r'.+//:[^/]/+.*', value)
```

Comment:
The `draft4_format_checker` uses the `rfc3987` module with format `uri`, which seems pretty permissive. It matches URIs like `urn:ISSN:1234-1234`. Are there technically invalid URIs we want to accept?